ts/RootContainer.tsx

```diff
@@ -83,6 +84,16 @@ class RootContainer extends React.PureComponent<Props> {
     AppState.addEventListener("change", this.handleApplicationActivity);
     // Hide splash screen
     SplashScreen.hide();
+    if (Platform.OS === "android" && Platform.Version >= 24) {
```

Comment:
This should read `Platform.Version >= 25`. Shortcuts where [introduced in Nougat](https://developer.android.com/reference/android/content/pm/ShortcutManager) (API level 25).

Maybe this can be extracted into an utility function somewhere to be reused, something along the lines of

```js
const androidSupportsShortcuts = () => { Platform.OS === "android" && Platform.Version >= 25 }
```

Reply:
cc @Edigio